README.md

```diff
@@ -0,0 +1,35 @@
+# Application Services Architecture
+
+This repository is intended to collect and keep up to date the proposals and decisions taken into the context of the Managed Application Services group.
+
+https://architecture.appservices.tech
+
+## Terminology cheat sheet
+
+ - `ADR` - Architecture Decision Record - records a decision (specific to *a single* service) that *MUST be followed* in applicable circumstances. The only way to avoid it is to supersede the decision.
+ - `PADR` - Platform Architecture Decision Record - records a decision (applying to *all* services) that *MUST be followed* in applicable circumstances. The only way to avoid it is to supersede the decision.
+ - `AP` - Architecture Proposal - records a decision that *SHOULD be taken into consideration* in applicable circumstances. When an AP becomes generally applicable it can be promoted to ADR/PADR.
+
+## Service Tech Leads
+
+The current Service Tech Leads are:
```

Comment:
In the past we wanted to avoid putting names to roles too explicitly. We could use a team for this. @emmanuelbernard wdyt?

Reply:
> We could use a team for this.

I cannot find a matching Team in GitHub:
https://github.com/orgs/bf2fc6cc711aee1a0c2a/teams
(but I can notice that offboarding is not working even in this org 🙁 )

can you suggest what to use here?

Reply:
One among those 4 is probably a good first guess:
https://github.com/orgs/bf2fc6cc711aee1a0c2a/teams/org-management-team

Reply:
@grdryn could you create a team for this?

Reply:
The group can now be annoyed as @bf2fc6cc711aee1a0c2a/service-leads (sorry ;D)

Reply:
fixed, thanks @grdryn for the support!